src/Roadmap/RoadmapNode.jsx

```diff
@@ -1,14 +1,80 @@
 import circle from "../assets/img/circle.png";
-
+import Tippy from "@tippyjs/react/headless";
+import short from "short-uuid";
+import { useSpring, animated } from "react-spring";
 const style = {
   position: "absolute",
   top: "calc(50% - 25px)",
   left: "calc(50% - 25px)",
   zIndex: 99,
 };
 
-const RoadmapNode = () => {
-  return <img src={circle} alt="level" style={style}/>
-}
+const tooltipStyle = {
+  backgroundColor: "black",
+  color: "white",
+  opacity: 0.8,
+  zIndex: 100,
+  width: "300px",
+  padding: "1rem",
```

Comment:
rem is good, but we have to stay consistent, either we use px or rem for all metrics